test/optional-validation.test.js

```diff
@@ -27,9 +30,10 @@ describe('optional-validation', function() {
       order: { type: Number, index: true, sort: true },
       vip: { type: Boolean },
     }, { forceId: true, strict: true });
-
-    db.automigrate(['User'], done);
-
+    var models = ['modelWithForceId', 'User'];
+    async.each(models, function(model, cb) {
+      db.automigrate(model, cb);
+    }, done);
```

Comment:
I believe you can pass multiple model names to automigrate function, no need to use `async.each`:

```js
db.automigrate(['modelWithForceId', 'User'], done);
```